### ISSUE TITLE
openapi3 cannot parse global tags

### ISSUE THREAD

**Issue by roosemberth** [Dorthu/openapi3#7]
Given the following yaml:

```yaml
openapi: 3.0.0
info:
  title: test
  version: '0.1'
  description: bar
components:
  links: {}
  schemas: {}
security: {}
tags:
  - name: ExampleTag
    description: foo
paths:
  /foo:
    summary: Example path
    get:
      tags:
        - ExampleTag
      responses:
        200:
          description: Successful operation.
          content:
            application/json:
              schema:
                type: string
```

expected output:
```shell
(venv) user@hostname:/work$ python3 -m openapi3 example.yaml
OK
```

actual output:
```
(venv) user@hostname:/work$ python3 -m openapi3 example.yaml
: Expected .tags to be one of [<class 'dict'>], got <class 'list'>

1 errors
```

version:  openapi3 0.0.2

**Comment by roosemberth** [comment]
I made a pr to fix this: https://github.com/Dorthu/openapi3/pull/8

**Comment by Dorthu** [comment]
merged #8 - this is fixed
